packages/a11y/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"tsBuildInfoFile": "../../.cache/a11y.tsbuildinfo"
```

Comment:
Is there an issue with using the default? Or conversely, is there value in having these all in the same top-level `.cache` directory?

Reply:
The default is would be `.tsbuildinfo` in the package root (`packages/a11y/.tsbuildinfo`). That's probably fine to use.

One bit of nuance is that it seems these files must be removed to regenerate the type declaration files. That means cleaning `build-types` should be accompanied by cleaning `.tsbuildinfo`.